documentation/modules/configuring/con-config-mirrormaker2-producers-consumers.adoc

```diff
@@ -0,0 +1,110 @@
+//
+// assembly-config-mirrormaker2.adoc
+
+[id='con-mirrormaker-producers-consumers-{context}']
+= Connector producer and consumer configuration
+
+[role="_abstract"]
+MirrorMaker 2.0 connectors uses internal producers and consumers.
+If needed, you can configure these producers and consumers to override the default settings. 
+
+For example, you can increase the `batch.size` for the source producer that sends topics to the target Kafka cluster to better accommodate large volumes of messages.
+
+IMPORTANT: Producer and consumer configuration options depend on the MirrorMaker 2.0 implementation, and may be subject to change.  
+
+The following tables describe the producers and consumers for each of the connectors and where you can add configuration. 
+
+.Source connector producers and consumers
+[cols="1,1a,3m",options="header"]
+|===
+
+|Type
+|Description
+|Configuration
+
+|Producer
+|Sends topic messages to the target Kafka cluster. Consider tuning the configuration of this producer when it is handling large volumes of data. 
+|mirrors.sourceConnector.config: producer.override.*
+
+|Producer
+|Writes to the `offset-syncs` topic, which maps the source and target offsets for replicated topic partitions. 
+|mirrors.sourceConnector.config: producer.*
+
+|Consumer
+|Retrieves topic messages from the source Kafka cluster.
+|mirrors.sourceConnector.config: consumer.* 
+|===
+
+.Checkpoint connector producers and consumers
+[cols="1,1a,3m",options="header"]
+|===
+
+|Type
+|Description
+|Configuration
+
+|Producer
+|Emits consumer offset checkpoints.
+|mirrors.checkpointConnector.config: producer.override.* 
+
+|Consumer
+|Loads the `offset-syncs` topic.
+|mirrors.checkpointConnector.config: consumer.*
+
+|===
+
+.Heartbeat connector producer
+[cols="1,1a,3m",options="header"]
+|===
+
+|Type
+|Description
+|Configuration
+
+|Producer
+|Emits heartbeats.
+|mirrors.heartbeatConnector.config: producer.override.*
+
+|===
+
+.Example configuration for connector producers and consumers
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaMirrorMaker2ApiVersion}
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker2
+spec:
+  version: {DefaultKafkaVersion}
+  # ...
+  mirrors:
+  - sourceCluster: "my-cluster-source"
+    targetCluster: "my-cluster-target"
+    sourceConnector:
+      tasksMax: 5
+      config:
+        producer.override.batch.size: 327680
+        producer.override.linger.ms: 5
+        producer.request.timeout.ms: 40000
+        consumer.fetch.max.bytes: 73400320
+        # ...
+    checkpointConnector:
+      config:
+        producer.override.request.timeout.ms: 40000
+        consumer.max.poll.interval.ms: 180000
+        # ...
+    heartbeatConnector:
+      config:
+        producer.override.request.timeout.ms: 40000
+        # ...      
+----
+
+NOTE: You can configure producers and consumers at the cluster level using `spec.clusters.config.producer.\*` or `spec.clusters.config.consumer.*` properties. 
```

Comment:
I think it would be better to remove it. 

`spec.cluster.config:producer.*` affects not only MirrorSourceConnector’s producer but also MirrorCheckpointConnector’s and MirrorHeartbeatConnector’s one, It can be described by explaining which producers are affected, however, it is somewhat hard to understand and there will be no use case where the same value is set for these three producers.

`spec.cluster.config:consumer.*` affects only three consumers, which load data in xxxBackingStore,  in Kafka Connect(, not xxxConnector) in MirrorMaker2. It is trivial to configure and does not need to be described.

Reply:
I removed this note after discussing with @tyamashi-oss